src/machine_helpers.py

```diff
@@ -28,3 +35,26 @@ def update_mongod_service(
     if role == Config.Role.CONFIG_SERVER:
         mongos_start_args = get_mongos_args(config, snap_install=True)
         add_args_to_env("MONGOS_ARGS", mongos_start_args)
+
+
+def setup_logrotate_and_cron() -> None:
+    """Create and write the logrotate config file."""
+    logger.debug("Creating logrotate config file")
+
+    with open("templates/logrotate.j2", "r") as file:
+        template = jinja2.Template(file.read())
+
+    rendered = template.render(
+        logs_directory=f"{MONGODB_COMMON_DIR}/{LOG_DIR}",
+        mongo_user=MONGO_USER,
+    )
+
+    with open("/etc/logrotate.d/mongodb", "w") as file:
+        file.write(rendered)
+
+    cron = (
+        "* 1-23 * * * root logrotate -f /etc/logrotate.d/mongodb\n"
+        "1-59 0 * * * root logrotate -f /etc/logrotate.d/mongodb\n"
+    )
```

Comment:
more noob questions from someone who doesn't understand cron, can you add comments explaining:
1. what each of these two are
2. the significance of `1-23` and `1-59`

Reply:
Let me actually simplify this